examples/auth_example.py

```diff
@@ -0,0 +1,33 @@
+"""
+auth_example.py
+
+Example script to demonstrate the usage of the app API in the CS3Client class.
+note that these are examples, and is not meant to be run as a script.
+
+Authors: Rasmus Welander, Diogo Castro, Giuseppe Lo Presti.
+Emails: rasmus.oscar.welander@cern.ch, diogo.castro@cern.ch, giuseppe.lopresti@cern.ch
+Last updated: 28/08/2024
+"""
+
+import logging
+import configparser
+from cs3client.cs3client import CS3Client
+
+config = configparser.ConfigParser()
+with open("default.conf") as fdef:
+    config.read_file(fdef)
+log = logging.getLogger(__name__)
+
+client = CS3Client(config, "cs3client", log)
+
+# Set client secret
+client.auth.set_client_secret("<your_client_secret_here>")
+# Checks if token is expired if not return ('x-access-token', <token>)
+# if expired, request a new token from reva
+auth_token = client.auth.get_token()
+
+# OR if you already have a reva token
+# Checks if token is expired if not return (x-access-token', <token>)
+# if expired, throws an AuthenticationException (so you can refresh your reva token)
+token = "<your_reva_token>"
+auth_token = client.auth.check_token(token)
```

Comment:
```suggestion
auth_token = cs3client.Auth.check_token(token)
```
maybe? to reinforce the fact that in this mode there's no need to instantiate an `auth` class.

Reply:
that's a good idea!

Reply:
Please import `Auth` and use it directly. This should not be part of the client object.